article.html

```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>der SPACES express | Artikel</title>
```

Comment:
this should be named "der SPACES express | article"

Reply:
Ist gefixt. Bitte nochmal reviewen